internal/server/iigointernal/orchestration.go

```diff
@@ -26,8 +26,11 @@ var monitoring monitor
 
 // RunIIGO runs all iigo function in sequence
 func RunIIGO(logger shared.Logger, g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.Client, gameConf *config.Config) (IIGOSuccessful bool, StatusDescription string) {
-	// featureJudge is an instantiation of the Judge interface
-	// with both the Base Judge features and a reference to client judges
+
+	removeDeadBodiesFromOffice(g)
```

Comment:
Best line of code ever !! ;)

Reply:
oh god seeing as we are simulating whole islands there must be a lot of bodies o_O